demo/src/vendor.ts

```diff
@@ -0,0 +1,6 @@
+import 'angular2/bundles/angular2-polyfills';
```

Comment:
It is not about trust. It was always working and everybody was doing it too without problems.

I will change it and write some scripts for deployment.